src/state/messages/convo/agent.ts

```diff
@@ -130,7 +131,7 @@ export class Convo {
 
   getSnapshot(): ConvoState {
     if (!this.snapshot) this.snapshot = this.generateSnapshot()
-    // logger.debug('Convo: snapshotted', {})
```

Comment:
We don't need this prefix, this logger instance has a `context` associated with it so they're already prefixed